app/src/main/res/values/strings.xml

```diff
@@ -305,11 +305,9 @@
     <string name="lbl_next_episode">Next Episode</string>
     <string name="lbl_album_name">Album name</string>
     <string name="lbl_song_title">Song Title</string>
-    <string name="pref_use_external_player_title">Use External Player for Videos</string>
-    <string name="pref_use_external_player_summary">Requires a capable app to play video types</string>
```

Comment:
These will need to be removed for all locales.

Reply:
Fixed, first time working on an android app with multiple languages so I forgot to remove the strings from other files. I noticed the string editor in android studio helps with that fortunately